circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  node:
+    version: 6.9
+  environment:
+    PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
```

Comment:
I think this is redundant - npm scripts (and I assume for compatibility, yarn scripts) are always run with `.bin` in the path.